### ISSUE TITLE
Voiceover says screen reader optimized mode disabled even if it is enabled

### ISSUE THREAD

**Issue by sandy081** [microsoft/vscode#194103]
Testing #193852

https://github.com/microsoft/vscode/assets/10746682/331f7b5a-4347-4ed5-97cf-efe35bfda644

**Comment by meganrogge** [comment]
Hmm, cannot reproduce...

**Comment by sandy081** [comment]
Here are the simple steps I followed to repro:

- Enable Voiceover
- Go to terminal and press Alt+f1
- Voiceover says to enable screen reader optimised more and press shift+option+f1 - this enables the screen reader optimised mode
- press escape to hide the terminal accessibility help
- press Alt+f1 again. Voiceover again say to enable the screen reader optimised mode.

**Comment by meganrogge** [comment]
I am still not reproducing. If you click in the editor, does it say that screen reader optimized mode is active or "editor is not accessible at this time"?

![Screenshot 2023-09-28 at 11 03 31 AM](https://github.com/microsoft/vscode/assets/29464607/d0d6dfb8-dce5-44ba-8db5-9be003c7a5b0)

**Comment by sandy081** [comment]
>  If you click in the editor, does it say that screen reader optimized mode is active or "editor is not accessible at this time"?

Does not say anything.

Probably it might be easy to sync directly?

**Comment by VSCodeTriageBot** [comment]
This issue has been closed automatically because it needs more information and has not had recent activity. See also our [issue reporting](https://aka.ms/vscodeissuereporting) guidelines.

Happy Coding!

**Comment by meganrogge** [comment]
Still not sure how you got into this state, can you reproduce?

**Comment by VSCodeTriageBot** [comment]
This issue has been closed automatically because it needs more information and has not had recent activity. See also our [issue reporting](https://aka.ms/vscodeissuereporting) guidelines.

Happy Coding!
